internal/k8s/volume_finder.go

```diff
@@ -96,6 +96,10 @@ func (f VolumeFinder) GetPersistentVolumes(ctx context.Context) ([]VolumeInfo, e
 			// powerflex will provide storagesystem id and powerstore will provide array IP
 			if info.StorageSystem == "" || len(info.StorageSystem) == 0 {
 				info.StorageSystem = volume.Spec.CSI.VolumeAttributes["arrayID"]
+				if info.StorageSystem == "" {
+					info.StorageSystem = volume.Spec.CSI.VolumeAttributes["arrayIP"]
+				}
```

Comment:
First powerstore changed to arrayID.
At line #98 we are already doing it. 
For Powerflex this should give us storage system (at line #87) `uid StorageSystem: volume.Spec.CSI.VolumeAttributes["StorageSystem"],`
Then at line #98 we should be able to get for Powerstore.
Line #99-101 not needed.